<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gearmotive Software</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="liveDataTab">
       <attribute name="title">
        <string>Live Data</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="PrimaryRow">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="Layout1">
            <item>
             <widget class="QLabel" name="Heading1">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Oil temperature ( °C )</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="OilTemp">
              <property name="font">
               <font>
                <pointsize>22</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Null</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="Layout2">
            <item>
             <widget class="QLabel" name="Heading2">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Primary axle ( RPM ) </string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="PrimaryAxleRpm">
              <property name="font">
               <font>
                <pointsize>22</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Null</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="Layout3">
            <item>
             <widget class="QLabel" name="Heading3">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Secondary axle ( RPM )</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="SecondaryAxleRpm">
              <property name="font">
               <font>
                <pointsize>22</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Null</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="Layout4">
            <item>
             <widget class="QLabel" name="Heading4">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Gearbox torque ( Nm )</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="GearboxTorque">
              <property name="font">
               <font>
                <pointsize>22</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Null</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="Layout5">
            <item>
             <widget class="QLabel" name="Heagin5">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Brake torque ( Nm )</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="BrakeTorque">
              <property name="font">
               <font>
                <pointsize>22</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Null</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="SecondaryRow">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="Layout6">
            <item>
             <widget class="QLabel" name="Heading6">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Air temperature ( °C )</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="AirTemp">
              <property name="font">
               <font>
                <pointsize>22</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Null</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="Layout7">
            <item>
             <widget class="QLabel" name="Heading7">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>asd</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="font">
               <font>
                <pointsize>22</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Null</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="Layout8">
            <item>
             <widget class="QLabel" name="Heading8">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>asd</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="font">
               <font>
                <pointsize>22</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Null</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="Layout9">
            <item>
             <widget class="QLabel" name="Heading9">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>asd</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="font">
               <font>
                <pointsize>22</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Null</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="Layout10">
            <item>
             <widget class="QLabel" name="Heading10">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>asd</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="font">
               <font>
                <pointsize>22</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Null</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="logViewerTab">
       <attribute name="title">
        <string>Log Viewer</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <widget class="QWidget" name="listContainerWidget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Anturit</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="sensorListWidget"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="chartContainerWidget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="InteractiveChartView" name="chartView" native="true"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="openLogFileButton">
                <property name="text">
                 <string>Avaa lokitiedosto...</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QSlider" name="timeSlider">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="valuesGroup">
              <property name="title">
               <string>Arvot valitulla ajankohdalla</string>
              </property>
              <layout class="QGridLayout" name="valuesLayout"/>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTiedosto">
    <property name="title">
     <string>&amp;Tiedosto</string>
    </property>
    <addaction name="actionStartLogging"/>
    <addaction name="actionStopLogging"/>
    <addaction name="separator"/>
    <addaction name="actionOpenLogFile"/>
   </widget>
   <widget class="QMenu" name="menuYhteydet">
    <property name="title">
     <string>&amp;Yhteydet</string>
    </property>
    <widget class="QMenu" name="menuBaud_rate">
     <property name="title">
      <string>Baudrate</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuPort_COM">
     <property name="title">
      <string>Port (COM)</string>
     </property>
    </widget>
    <addaction name="menuBaud_rate"/>
    <addaction name="menuPort_COM"/>
    <addaction name="separator"/>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <addaction name="menuTiedosto"/>
   <addaction name="menuYhteydet"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDisconnect">
   <property name="text">
    <string>Katkaise yhteys</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Yhdistä</string>
   </property>
  </action>
  <action name="actionStartLogging">
   <property name="text">
    <string>&amp;Aloita lokitus...</string>
   </property>
  </action>
  <action name="actionStopLogging">
   <property name="text">
    <string>&amp;Lopeta lokitus</string>
   </property>
   <property name="iconText">
    <string>Lopeta lokitus</string>
   </property>
  </action>
  <action name="actionOpenLogFile">
   <property name="text">
    <string>&amp;Avaa lokitiedosto...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>QtCharts/QChartView</header>
  </customwidget>
  <customwidget>
   <class>InteractiveChartView</class>
   <extends>QChartView</extends>
   <header>interactivechartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
